src/VirtualClient/VirtualClient.Actions/OpenSSL/OpenSslMetricsParser.cs

```diff
@@ -122,7 +128,13 @@ public override IList<Metric> Parse()
                 this.RSAResults = rsaResults;
             }
 
-            if (!cipherResultsValid && !rsaResultsValid)
+            if (!rsaResultsValid & this.TryParseOpsPerformanceResults(out DataTable opsResults))
```

Comment:
Please merge with main because I renamed rsa to signVerifyResults

Reply:
Done